docs(discussion/*): fix grammar (#7816)
remix-run/remix: docs/discussion/concurrency.md

submission 3:             |-----✓-----✅
```

> Because the revalidation from submission (2) started later and landed earlier than submission (1), the requests from submission (1) are canceled and only the data from submission (2) is committed to the UI. It was requested later so it's more likely to contain the updated values from both (1) and (2).

Reviewer: Double `l` is fine as well, so let's keep that

```suggestion
Because the revalidation from submission (2) started later and landed earlier than submission (1), the requests from submission (1) are cancelled and only the data from submission (2) is committed to the UI. It was requested later so it's more likely to contain the updated values from both (1) and (2).
```

## Potential for Stale Data

> It's unlikely your users will ever experience this, but there are still chances for the user to see stale data in very rare conditions with inconsistent infrastructure. Even though Remix cancels requests for stale data, they will still end up making it to the server. Canceling a request in the browser simply releases browser resources for that request, it can't "catch up" and stop it from getting to the server. In extremely rare conditions, a canceled request may change data after the interrupting `action`'s revalidation lands. Consider this diagram:

Reviewer: ```suggestion
It's unlikely your users will ever experience this, but there are still chances for the user to see stale data in very rare conditions with inconsistent infrastructure. Even though Remix cancels requests for stale data, they will still end up making it to the server. Cancelling a request in the browser simply releases browser resources for that request, it can't "catch up" and stop it from getting to the server. In extremely rare conditions, a cancelled request may change data after the interrupting `action`'s revalidation lands. Consider this diagram:
```

        <ComboboxInput
          name="q"
          onChange={(event) =>

>             //Submit the form onChange to get the list of cities

Reviewer: ```suggestion
            // Submit the form onChange to get the list of cities
```
